workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/common/utils/constants/filters.ts

```diff
@@ -6,16 +6,17 @@
 import {getCamelCase} from '../getCamelCase';
 
 const todayDate = new Date();
+
 const currentYear = todayDate.getFullYear();
 
-const currentFiscalYearEnd = `${currentYear}-12-31`;
-const currentFiscalYearStart = `${currentYear}-01-01`;
-const previousFiscalYearStart = `${currentYear - 1}-01-01`;
+export const currentFiscalYearEnd = `${currentYear}-12-31`;
+export const currentFiscalYearStart = `${currentYear}-01-01`;
+export const previousFiscalYearStart = `${currentYear - 1}-01-01`;
 
 const fiscalYearFilterCloseDate = `closeDate ge ${currentFiscalYearStart} and closeDate le ${currentFiscalYearEnd}`;
 const fiscalYearFilterCreatedDate = `createdDate ge ${currentFiscalYearStart}T00:00:00Z and createdDate le ${currentFiscalYearEnd}T23:59:59Z`;
 const fiscalYearFilterSubmitDate = `submitDate ge ${currentFiscalYearStart}T00:00:00Z and submitDate le ${currentFiscalYearEnd}T23:59:59Z`;
-const previousToCurrentYearFilterSubmitDate = `submitDate ge ${previousFiscalYearStart}T00:00:00Z and submitDate le ${currentFiscalYearEnd}T23:59:59Z`;
+const previousToCurrentYearFilterSubmitDate = `submitDate ge 2023-01-01T00:00:00Z and submitDate le ${currentFiscalYearEnd}T23:59:59Z`;
```

Comment:
note that this is being used in other filters, we cant make this change here, otherwise other filters will have the 2023 date, and it only appiles to the submitted deal registrations.